src/Workspaces/CoreTest/CodeStyle/EditorConfigCodeStyleParserTests.cs

```diff
@@ -56,5 +62,49 @@ static void TestParseEditorConfigCodeStyleOption(string args, bool isEnabled, Re
             Assert.True(result.Notification.Severity == severity,
                         $"Expected {nameof(severity)} to be {severity}, was {result.Notification.Severity}");
         }
+
+        [Theory]
+        [InlineData("never:none", (int)AccessibilityModifiersRequired.Never, ReportDiagnostic.Suppress)]
+        [InlineData("always:suggestion", (int)AccessibilityModifiersRequired.Always, ReportDiagnostic.Info)]
+        [InlineData("for_non_interface_members:warning", (int)AccessibilityModifiersRequired.ForNonInterfaceMembers, ReportDiagnostic.Warn)]
+        [InlineData("omit_if_default:error", (int)AccessibilityModifiersRequired.OmitIfDefault, ReportDiagnostic.Error)]
+
```

Comment:
I am torn between "ugh, why do you have a blank line in a huge block of attributes?????" and "oh, that's a nice way to group them".

Reply:
I felt the same way until the `[WorkItem]` attribute made such a nice section header.